tests/integration/local/invoke/test_integrations_cli.py

```diff
@@ -233,6 +234,8 @@ def test_invoke_with_docker_network_of_host(self):
 
         self.assertEquals(return_code, 0)
 
+    @skipIf(platform.system().lower() == 'windows',
```

Comment:
Can you instead add a separate decorator to a testutils somewhere? I expect many such cases need exclusion for Windows. I'd rather have it defined in one place.

Reply:
Ahh. Good call. Will update this to make it reusable.